distribution/server/src/assemble/LICENSE.bin.txt

```diff
@@ -642,3 +645,11 @@ interchange format, which can be obtained at:
    * licenses/LICENSE-protobuf.txt (New BSD License)
  * HOMEPAGE:
 * https://github.com/google/protobuf
+
+Contains a modified portion of and optionally depends on 'Protocol Buffers', Google's data
```

Comment:
is this due to Oauth0 ? in this case please add a reference

Reply:
whoops, that was a mistake. Sorry